src/test/scala/tech/sourced/gemini/CassandraSparkSpec.scala

```diff
@@ -241,10 +245,65 @@ class CassandraSparkSpec extends FlatSpec
 
   "Hash with limit" should "collect files only from limit repos" in {
     val gemini = Gemini(sparkSession)
-    val repos = gemini.hash("src/test/resources/siva", 1).select(Gemini.meta.repo).distinct().count()
+    val repos = gemini.hash("src/test/resources/siva", 1).select("repository_id").distinct().count()
```

Comment:
Why is it needed to stop using the variable in favor of the constant?

Ideally, we should not have any string literals in for the fields and have `table["meta"].repo`, `table["hashtable"].something`, etc. This simplifies refactorings and increase maintainability.

It most probably will be done as a separate PR with refactoring later on, but until then, if there is no strong reason, I suggest to keep using this var at least for a 'meta' table.

Reply:
because now hash doesn't return DF for database anymore. But original DF of engine spark. We have to keep original names to be able to call UDFs.